nf_core/components/create.py

```diff
@@ -136,20 +139,36 @@ def create(self):
         # Check existence of directories early for fast-fail
         self.file_paths = self._get_component_dirs()
 
-        if self.component_type == "modules":
-            # Try to find a bioconda package for 'component'
-            self._get_bioconda_tool()
+        if self.migrate_pytest:
+            # Rename the component directory to old
+            component_old = self.component_dir + "_old"
+            component_old_path = Path(self.directory, self.component_type, self.org, component_old)
+            Path(self.directory, self.component_type, self.org, self.component_dir).rename(component_old_path)
+        else:
+            if self.component_type == "modules":
+                # Try to find a bioconda package for 'component'
+                self._get_bioconda_tool()
 
-        # Prompt for GitHub username
-        self._get_username()
+            # Prompt for GitHub username
+            self._get_username()
 
-        if self.component_type == "modules":
-            self._get_module_structure_components()
+            if self.component_type == "modules":
+                self._get_module_structure_components()
 
         # Create component template with jinja2
         self._render_template()
         log.info(f"Created component template: '{self.component_name}'")
 
+        if self.migrate_pytest:
+            self._copy_old_files(component_old_path)
+            log.info("Migrate pytest tests: Copied original module files to new module")
+            try:
+                self._update_nftest_file()
+                log.info("Migrate pytest tests: Updated `main.nf.test` with contents of pytest")
+            except Exception as e:
+                log.info(f"Could not update `main.nf.test` file: {e}")
+            shutil.rmtree(component_old_path)
```

Comment:
```suggestion
               shutil.rmtree(component_old_path)
            except Exception as e:
                log.info(f"Could not update `main.nf.test` file: {e}")
```
So we don't lose something by accident on error. or you think git will help out anyway and this is a cleaner approach?